src/Messages/Interactions/TSIncomingMessage.h

```diff
@@ -84,8 +68,8 @@ extern NSString *const TSIncomingMessageWasReadOnThisDeviceNotification;
 
 
 /**
- * For sake of a smaller API, you must specify an author id for all incoming messages
- * though we technically could get the author id from a contact thread.
+ * For sake of a smaller API, and simplifying assumptions elsewhere, you must specify an author id for *all* incoming
+ * messages, even though we technically could infer the author id for a contact threads.
```

Comment:
nit: extra preposition?
`...the author id for contact threads.`
or perhaps 
`...the author id of contact threads.`
Are contact threads 1:1 conversations?  Making sure I get the nomenclature.